src/FixBookControl.java

```diff
@@ -19,7 +19,7 @@ public void Set_Ui(FixBookUI ui) {
 			throw new RuntimeException("FixBookControl: cannot call setUI except in INITIALISED state");
 		}	
 		this.UI = ui;
-		ui.Set_State(FixBookUI.UI_STATE.READY);
+		ui.Set_State(FixBookUI.UIState.READY);

```

Comment:
Can we clarify whether we are using camelBack style - Should it be UIState or UiState?

Reply:
'UI' is always capitalised. This is a rule we verbally decided on in-meeting and again discussed on the Discord message thread.
I am currently creating an updated code style guidelines document on Google Drive, to ensure the rules are all in one place.